src/utils/iterateTabbable.ts

```diff
@@ -57,19 +57,29 @@ export function* iterateTabbableElements(
  * @param strict
  */
 export function isTabbable(elem: HTMLElement, strict = false): boolean {
+  // Always tabbable if tabindex is explicitly 0 or higher
+  const tabIndexAttribute = elem.getAttribute('tabindex')
```

Comment:
I wasn't sure so I doubled checked in dev tools, it looks like if you explicitly set a tab index of 0 on an element, then set the disabled attribute, the tab index still comes back as 0. 
<img width="495" alt="image" src="https://user-images.githubusercontent.com/8960591/110552311-f4f07000-80eb-11eb-806f-c87f49762b1a.png">

In this case, we'd return true though, even if `disabled` is set. I don't think we want to return early here, instead we should probably move lines 66 to 77 above this check and return early if any of the not tabbable checks are true

Reply:
Oh wow, so `disabled` trumps `tabindex="0"`, eh? That seems bad. We should probably avoid and advise against using the `disabled` attribute on form elements, instead opting for `aria-disabled` and relevant styling/behavior. Will fix.

ARIA actually recommends allowing disabled buttons to be focusable in some situations, which is why I assumed `tabindex="0"` would win here. See: https://www.w3.org/TR/wai-aria-practices-1.1/#kbd_disabled_controls

Reply:
From what I understand, the tab index attribute returned from `getAttribute` doesn't change ever if the user sets it explicitly. So if someone had an element that they wanted to make focusable but might dynamically set it to be disabled, hidden, or add/remove the `href`, `getAttribute("tabindex")` is always going to return `0`. I tested it out in the browser with `hidden` and missing `href`.  Because of that, I think this check needs to be moved to the bottom of the file because I could definitely see a use case where someone needs to make something focusable by default, but it's possible that it might become unfocusable/disabled/hidden later.